frontend/src/app/components/settings/settings.component.html

```diff
@@ -0,0 +1,69 @@
+<div *ngIf="isLoading === true" class="narrow-body" style="margin-top: 50px; width: 100%; align-items: center; justify-content: center">
```

Comment:
this layout doesn't really work on mobile
![image](https://user-images.githubusercontent.com/40121408/107050661-7ecdb600-6799-11eb-80f7-bffa44cc5c42.png)